### DISCUSSION TITLE
KusionStack + KCL 11.23 中文周会

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#11] [General]
腾讯会议列表：778-2381-6338

**Comment by OliverPerezzz** [comment]
Sembra una conferenza tecnica piuttosto formale. La combinazione di KusionStack e KCL è intrinsecamente attraente e dovrebbe offrire molti contenuti pratici da condividere. Una conferenza in lingua cinese è anche più intuitiva per gli sviluppatori nazionali, facilitando una comunicazione più efficiente. Avere un programma o informazioni sui relatori sarebbe ancora meglio, consentendo ai partecipanti di preparare le domande in anticipo. Speriamo che questa conferenza si concentri maggiormente su casi di studio concreti piuttosto che su semplici introduzioni concettuali. Caspero Casino porta il divertimento del [casperocasinos.it](https://casperocasinos.it/) casinò direttamente a casa tua. Nel complesso, vale sicuramente la pena prestarle attenzione.
